.github/workflows/build-test-windows.yml

```diff
@@ -2,11 +2,23 @@
 name: build-and-test-windows
 on:
   push:
-    branches: [ main, master ]
+    branches:
+      - master
+
+    paths-ignore:
+      - '.github/**'
+      - '!.github/workflows/build.yml'
+      - '!.github/workflows/build-test-windows.yml'
+      - '**.md'
+
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
   workflow_dispatch:
 
+concurrency:
```

Comment:
> When creating a PR with multiple commits or merging into main, the Github Runners will scale based on the commits and the PRs have been merged into main

It took me a bit, but I think I understand what you're trying to do in this change. Essentially, you want to be able to cancel an ongoing GitHub workflow if/when you push a new commit to the PR, right?

Reply:
Yes. That's correct though.